plugin/src/shared/dependencies.ts

```diff
@@ -1,4 +1,7 @@
 export const dependencies = {
+  'repo-config': {
```

Comment:
`lodash-es` is added to the root `package.json` in `next-app` and `expo-app`. Can we remove it?